.travis.yml

```diff
@@ -8,17 +8,27 @@ script: skip
 
 jobs:
     include:
-        - stage: test
-          php: 7.2
-          name: Run bin
-          script:
-              - bin/CIConfigGen.php
+        -
+            stage: test
+            php: 7.2
+            name: Run bin
+            script:
+                - bin/CIConfigGen.php
+                - bin/CIConfigGen.php craft:generate composer.json
 
-        - stage: test
```

Comment:
This syntax is bit hacked yaml. It might break on some systems, so better no use it

```yaml
- x: y 
```